### ISSUE TITLE
Documentation mistake for Image.crop()

### ISSUE THREAD

**Issue by victorzheng1** [python-pillow/Pillow#5271]
box should be left, BOTTOM, right, TOP in the implementation versus what the documentation currently says.

**Comment by radarhere** [comment]
For the record, https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.crop
> Returns a rectangular region from this image. The box is a 4-tuple defining the left, upper, right, and lower pixel coordinate. See Coordinate System.

If I run the following code on one of our sample images - https://github.com/python-pillow/Pillow/blob/master/Tests/images/hopper.png - where 0 should be upper, and 64 should be lower.
```python
from PIL import Image
im = Image.open("Tests/images/hopper.png")
im.crop((0, 0, 128, 64)).show()
```
I get the top half of the image as expected.
![cropped](https://user-images.githubusercontent.com/3112309/108338062-f5eb5b80-7229-11eb-98c3-8888e5a80ff2.png)

So this fits with the documentation, meaning that I don't have the same experience as you. Could you provide some code to demonstrate what you're seeing?

**Comment by victorzheng1** [comment]
Based on my own experience, I think there should be more clarification on what the lower, upper, etc. mean.  Lower and upper have counterintuitive definitions. The "lower" y coordinate has a higher value than the "upper" y coordinate and vice versa.  Not the biggest issue, but I think it helps if there was more clarification or a change of terms.

**Comment by radarhere** [comment]
If it helps, the docs say ["See Coordinate System"](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.crop). That then [reads](https://pillow.readthedocs.io/en/stable/handbook/concepts.html#coordinate-system)
> The Python Imaging Library uses a Cartesian pixel coordinate system, with (0,0) in the upper left corner. Note that the coordinates refer to the implied pixel corners; the centre of a pixel addressed as (0, 0) actually lies at (0.5, 0.5).
>
> Coordinates are usually passed to the library as 2-tuples (x, y). Rectangles are represented as 4-tuples, with the upper left corner given first. For example, a rectangle covering all of an 800x600 pixel image is written as (0, 0, 800, 600).

**Comment by victorzheng1** [comment]
Yes, I understand that. Just seems less user-friendly with the naming imo, but this was all I wanted to voice.
